code/datums/keybinding/xeno.dm

```diff
@@ -441,12 +443,14 @@
 	full_name = "Hunter: Mark"
 	description = "Mark that lonely marine so that you can track with Psychic Trace."
 	keybind_signal = COMSIG_XENOABILITY_HUNTER_MARK
+	hotkey_keys = list("F")
```

Comment:
I will not be surprised if players hate me for changing up their keybind, but in the long run, this is for the new player trying out the caste.